packages/data-context/src/data/index.ts

```diff
@@ -1,7 +1,9 @@
 /* eslint-disable padding-line-between-statements */
 // created by autobarrel, do not modify directly
 
+export * from './CypressEnv'
```

Comment:
Just curious, do we need to expose these?

Reply:
These get generated automatically